exporters/ibmmq/e2e/entrypoint-consumer.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+while true; do
+    echo ${MQ_APP_PASSWORD} | /opt/mqm/samp/bin/amqsgetc DEV.QUEUE.1 QM1
```

Comment:
Do not we need to add a `sleep` here? I think this loop is going to spam the MQ and every core available.

Reply:
Oh, it is a consumer, is blocking. Nevermind.